esmvaltool/cmorizers/obs/cmorize_obs_grace.py

```diff
@@ -0,0 +1,170 @@
+"""ESMValTool CMORizer for GRACE.
+
+Tier
+   Tier 3
+Source
+   https://podaac.jpl.nasa.gov/dataset/TELLUS_GRAC-GRFO_MASCON_CRI_GRID_RL06_V2
+Last access
+   20200612
+
+Download and processing instructions
+ - Go to the above link
+ - Click the tab "Data Access"
+ - Log in with Earthdata account
+ - Downloaded the following files:
+     - CLM4.SCALE_FACTOR.JPL.MSCNv02CRI.nc
+     - GRCTellus.JPL.200204_202004.GLO.RL06M.MSCNv02CRI.nc
+     - LAND_MASK.CRI.nc
+
+
+Modification history
+   20200630-crezee_bas: written.
+"""
+
+import logging
+import os
+import urllib
+from cf_units import Unit
+from copy import deepcopy
+from datetime import datetime
+
+import iris
+import numpy as np
+import pandas as pd
+import xarray as xr
+from dateutil import relativedelta
+
+from esmvalcore.preprocessor import regrid_time
+from esmvaltool.cmorizers.obs import utilities as utils
+
+logger = logging.getLogger(__name__)
+
+
+def _make_monthly_data_contiguous(in_file, out_file, raw_varname, cfg):
+    original = xr.open_dataset(in_file)[raw_varname]
+
+    with urllib.request.urlopen(cfg['grace_table']) as response:
+        grace_months_table = pd.read_csv(response)
```

Comment:
Maybe it would be more consistent to either download all files automatically, or all files manually? Because now it looks like you download this file automatically and ask the user to download the rest manually?

Reply:
Yes, that is true. Changed it to manual download, it is also good if the user checks the months table since there could be errors.